modules/installing-rhv-setting-up-ca-certificate.adoc

```diff
@@ -14,25 +14,33 @@ Later, you provide the certificate to the installation program.
 
 .Procedure
 
-. Download the CA certificate using either of these methods:
-** Go to the Manager's webpage and click the *CA Certificate* link. The URL of the Manager's webpage uses this format: `https://<engine-fqdn>/ovirt-engine/`.
+. Use either of these two methods to download the CA certificate:
+** Go to the Manager's webpage, `https://<engine-fqdn>/ovirt-engine/`. Then, under *Downloads*, click the *CA Certificate* link.
 ** Run the following command:
 +
 ----
 $ curl -k 'https://<engine-fqdn>/ovirt-engine/services/pki-resource?resource=ca-certificate&format=X509-PEM-CA' -o /tmp/ca.pem  <1>
 ----
 <1> For `<engine-fqdn>`, specify the fully qualified domain name of the {rh-virtualization} Manager, such as `rhv-env.virtlab.example.com`.
 
-. Add the certificate to the computer using the appropriate method for your operating system:
-** For Linux, copy the CA certificate to the directory for server certificates, and update the CA trust. For example:
+. For Linux, copy the CA certificate to the directory for server certificates
 +
 ----
 $ sudo cp /tmp/ca.pem /etc/pki/ca-trust/source/anchors/ca.pem
-$ update-ca-trust
 ----
+. Configure the CA file to grant rootless user access to the Manager. Set the CA file permissions to have an octal value of `0644` (symbolic value: `-rw-r--r--`). For example, run:
 +
+----
+$ sudo chmod 0644 /tmp/ca.pem /etc/pki/ca-trust/source/anchors/ca.pem
+----
+. Add the certificate to the computer using the appropriate method for your operating system:
```

Comment:
s/using/by using

Maybe s/Add/Apply or a clarification to "computer" - it might just be me, but it scanned like you'd already added the cert to a Linux computer, and I don't want this to be missed.

Reply:
Thanks. I've updated the phrase to be clearer. Now it says `Add the certificate to the certificate manager for your operating system:`